servicetalk-traffic-resilience-http/src/main/java/io/servicetalk/traffic/resilience/http/AbstractTrafficResilienceHttpFilter.java

```diff
@@ -372,9 +371,20 @@ private void signal(final int newSignal) {
         public String toString() {
             return "TrackingDelegatingTicket{" +
                     "delegate=" + delegate +
-                    ", requestHashCode=" + requestHashCode +
+                    ", requestHashCode=" + parentRequest.hashCode() +
+                    ", requestHost=" + parentRequest.host() +
                     ", signaled=" + signaled +
                     '}';
         }
+
+        // TODO: this should be removed after we're confident we're not leaking tickets.
+        @Override
+        protected void finalize() throws Throwable {
```

Comment:
We could also make this an additional wrapper that we add conditionally based on a flag. If this code already has broad usage that might be a better idea.